src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.targets

```diff
@@ -27,6 +27,7 @@
   <Import Project="Microsoft.Android.Sdk.DefaultProperties.targets" />
   <Import Project="$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Common.Debugging.props"
       Condition="Exists('$(MSBuildThisFileDirectory)..\tools\Xamarin.Android.Common.Debugging.props') And '$(DesignTimeBuild)' != 'true' "/>
-  <Import Project="Microsoft.Android.Sdk.NativeAOT.targets" Condition=" '$(_AndroidRuntime)' == 'NativeAOT' And '$(DesignTimeBuild)' != 'true' " />
+  <Import Project="Microsoft.Android.Sdk.CoreCLR.targets"   Condition=" '$(_AndroidRuntime)' == 'CoreCLR' " />
+  <Import Project="Microsoft.Android.Sdk.NativeAOT.targets" Condition=" '$(_AndroidRuntime)' == 'NativeAOT' " />
```

Comment:
Just curious, why was there: `'$(DesignTimeBuild)' != 'true'` before ?

Reply:
I think this was accidental in an unrelated change: https://github.com/dotnet/android/commit/9ed1469303b9c842cfd76cfb8d2b04dd1fd7255f

When I saw it, felt like it would be wrong because it would prevent NativeAOT packs from being used in a design-time build.